### PR TITLE
Add reducer action

### PR DESCRIPTION
**Pull request by brhanuhailu**
# In this pull request I:
- [X] Added Redux Toolkit to the existing React app
- [X] Created a separate redux folder then add books/booksSlice.js, categories/categoriesSlice.js, and Store.js files inside it.
- [X] Create actions, initial state, and reducers for each file.
- [X] Combine both reducers into a root reducer by using the configureStore function.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @brhanuhailu,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive README. Kindly format your README using [the template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by brhanuhailu** [comment]
@TedLivist  Ohh Thank you very much for your constructive feedback

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (danmainah) [inline src/redux/store.js]: - [ ] @brhanuhailu its not good practice to disable the linter check for your file, kindly remove the disable and instead fix the linter errors, linter checks helped us write clean and better code.

File: src/index.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (danmainah) [inline src/index.js]: - [ ] @brhanuhailu it's not good practice to disable the linter check for your file, kindly remove the disable and instead fix the linter errors, linter checks helped us write clean and better code.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @brhanuhailu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 2 above)
Comment (brhanuhailu) [reply src/index.js]: @danmainah  thank you very much solved

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @brhanuhailu , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brhanuhailu** [comment]
@VICTOR-CHUKWU  Thank you for your time

**Pull request merged**